src/download.ts

```diff
@@ -7,11 +7,46 @@ import { Browser, Page } from 'puppeteer';
 let SAVE_DESTINATION = '';
 const SAVE_AS: string = config.get('saveAs');
 const MULTI_LANGUAGE: boolean = config.get('multiLanguage');
+let courseUrlSlugList = [];
```

Comment:
Make it const and local variable of `fetchAllCoursesAvailableToDownload()`.

Reply:
Can make it constant but it can't be moved inside `fetchAllCoursesAvailableToDownload()` because it is a recursive function and it will be re-initialized in that case, which we don't want.